Sources/SwiftProtobuf/UnknownStorage.swift

```diff
@@ -17,7 +17,7 @@
 import Foundation
 
 public struct UnknownStorage: Equatable {
-  internal var data = Data()
+  public private(set) var data = Data()
```

Comment:
I'm gonna go with private for the moment because we don't actually want to force set the data, it should be always coming in via `append()`